src/frontend/components/UploadManager/index.tsx

```diff
@@ -58,6 +59,7 @@ export const UploadManager = ({ children }: React.PropsWithChildren<{}>) => {
             file!.name,
             file!.type,
           );
+          await getResource(objectType, objectId);
```

Comment:
No need to update `UploadState` if this request failed?

Reply:
No, it should stay in pending. It's the upload manager job to manage the error and inform the component using it.